packages/react-core/src/components/Menu/MenuContent.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/Menu/menu';
+import { css } from '@patternfly/react-styles';
+
+export interface MenuContentProps extends React.HTMLProps<HTMLElement> {
+  /** Items within group */
+  children?: React.ReactNode;
+  /** Forwarded ref */
+  innerRef?: React.Ref<any>;
+}
+
+const MenuContentBase: React.FunctionComponent<MenuContentProps> = ({ children }: MenuContentProps) => (
+  <div className={css(styles.menuContent)}>{children}</div>
+);
+
+export const MenuContent = React.forwardRef((props: MenuContentProps, ref: React.Ref<HTMLElement>) => (
+  <MenuContentBase {...props} innerRef={ref} />
+));
```

Comment:
You are not doing anything with the `innerRef` prop, this way we'd just block the ref forwarding. Plus any additional props will be ignored and for instance custom class would be impossible to add. You can also simplify this by using the `div` directly
```suggestion
export const MenuContent = React.forwardRef((props: MenuContentProps, ref: React.Ref<HTMLElement>) => (
  <div {...props} className={css(styles.menuContent, props.className)} ref={ref} />
));
```